python/mlx/nn/losses.py

```diff
@@ -532,3 +532,58 @@ def cosine_similarity_loss(
     loss = mx.sum(x1 * x2, axis=axis) / mx.maximum(x1_norm * x2_norm, eps)
 
     return _reduce(loss, reduction)
+
+
+def margin_ranking_loss(
+    inputs1: mx.array,
+    inputs2: mx.array,
+    targets: mx.array,
+    margin: float = 0.0,
+    reduction: Reduction = "mean",
```

Comment:
Oh sorry for just seeing that. All our losses have `none` as the default reduction. Could you maybe change that too.

Reply:
OK, I changed it!